src/components/TopBar.js

```diff
@@ -71,8 +71,8 @@ const styles = {
 const betaTag = b({
   style: {
     fontSize: 8, lineHeight: '9px',
-    color: 'white', backgroundColor: colors.brick,
-    padding: '3px 5px', marginLeft: '0.4rem', verticalAlign: 'middle',
+    color: 'white', backgroundColor: '#73AD43',
```

Comment:
Is this color a one-off?

Reply:
Probably not, but we don't have the updated style guide yet so I figured to do it this way until then.